scripts/release/update-comment.js

```diff
@@ -1,27 +1,35 @@
-#!/usr/bin/env node
-
 const fs = require('fs');
-const glob = require('glob');
 const proc = require('child_process');
+const semver = require('semver');
+const run = (cmd, ...args) => proc.execFileSync(cmd, args, { encoding: 'utf8' }).trim();
```

Comment:
Added this helper. `encoding: 'utf8'` makes `.toString()` unnecessary. `.trim()` makes `.filter(Boolean)` unnecessary.

Reply:
nice !